src/main/java/com/synopsys/integration/alert/common/descriptor/config/field/DropDownConfigField.java

```diff
@@ -23,24 +23,24 @@
  */
 package com.synopsys.integration.alert.common.descriptor.config.field;
 
-import java.util.List;
+import java.util.Collection;
 
 import com.synopsys.integration.alert.common.enumeration.FieldGroup;
 import com.synopsys.integration.alert.common.enumeration.FieldType;
 
 public class DropDownConfigField extends ConfigField {
```

Comment:
Change the name of this field to SelectConfigField and add a boolean for searchable and a boolean for multiSelect